frontend/src/pages/login/login.css

```diff
@@ -0,0 +1,41 @@
+.container {
```

Comment:
If we really use them we must make them global and use them across all components

Reply:
I'll need to take a look at that and refactor later so that we can use them everywhere. I'd leave that as is for now.